scripts/prerequisite_packages.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+
+set -e
+
+#Enter kubernetes version
+v='1.21.1-00'
+
+ip_tables(){
+   echo '*****LETTING IPTABLES SEE BRIDGED TRAFFIC*****'
+   sudo modprobe br_netfilter
+   sudo apt-get -y update
+   echo -e 'br_netfilter' | cat > /etc/modules-load.d/k8s.conf
+   echo -e 'net.bridge.bridge-nf-call-ip6tables = 1\nnet.bridge.bridge-nf-call-iptables = 1' | cat >> /etc/sysctl.d/k8s.conf
+   sysctl --system
+}
+
+docker_install(){
+   echo '*****INSTALLING DOCKER*****'
+   sudo apt-get update -y 
+   if [ "$(which docker)" != "" ]
+    then
+       echo "Docker is already installed"
+    else
+       sudo apt-get install docker.io -y
+   fi
+}
+
+kube_packages(){
+   echo '*****INSTALLING KUBEADM, KUBELET AND KUBECTL*****'
+   sudo apt-get update -y
+   if [ "$(which apt-transport-https ca-certificates curl)" != "" ]
+    then
+      echo "apt-transport-https ca-certificates curl is already installed"
+    else
+      sudo apt-get install apt-transport-https ca-certificates curl -y
+   fi
+   sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+   echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+   sudo apt-get update 
+   sudo apt-get install -y kubelet=$v kubectl=$v kubeadm=$v
+   sudo apt-mark hold kubelet kubeadm kubectl 
+   systemctl start docker.service
+   systemctl enable docker.service
+}
+
+kube_cluster(){
+   echo '*****STARTING CLUSTER USING KUBEADM*****'
+   echo y | kubeadm reset # To remove the existing cluster if running
+   sudo rm -rf $HOME/.kube/config
+   kubeadm init
+   systemctl restart kubelet
+   export KUBECONFIG=/etc/kubernetes/admin.conf
+   export kubever=$(kubectl version | base64 | tr -d '\n') 
+   kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
+   mkdir -p $HOME/.kube
```

Comment:
why need 55 line to 57 line ("mkdir -p $HOME/.kube", "sudo cp /etc/kubernetes/admin.conf $HOME/" and  "sudo chown $(id -u):$(id -g) $HOME/admin.conf". could you give a explain?